lib/matplotlib/pyplot.py

```diff
@@ -1500,65 +1500,59 @@ def axis(*v, **kwargs):
 
 def xlabel(s, *args, **kwargs):
     """
-    Set the *x* axis label of the current axis.
+    Set the x-axis label of the current axes.
 
-    Default override is::
-
-      override = {
-          'fontsize'            : 'small',
-          'verticalalignment'   : 'top',
-          'horizontalalignment' : 'center'
-          }
+    Call signature::
 
-    .. seealso::
+        xlabel(label, fontdict=None, labelpad=None, **kwargs)
 
-        :func:`~matplotlib.pyplot.text`
-            For information on how override and the optional args work
+    This is just a shortcut for calling `.set_xlabel` on the current axes.
```

Comment:
I feel like this implies that either `ax.xlabel` or `plt.set_xlabel valid`.
Alternative suggestion is something like: this is the pyplot interface equivalent of calling :method:`~matplotlib.axes.Axes.set_xlabel`on the current axes.